src/appleseed/renderer/kernel/rendering/pixelrendererbase.cpp

```diff
@@ -154,21 +92,20 @@ void PixelRendererBase::on_pixel_end(
     const AABB2i&               tile_bbox,
     AOVAccumulatorContainer&    aov_accumulators)
 {
-    aov_accumulators.on_pixel_end(pi);
+    static const size_t MaxWarningsPerThread = 5;
 
-    // todo: mark pixel as faulty in the diagnostic map.
```

Comment:
If I understand correctly this todo is done because we now have a dedicated Invalid Samples AOV?

Reply:
yes